servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -382,7 +380,13 @@ def _get_base_url():
 def _format_query_labels(orig_query_labels):
     if orig_query_labels is None:
         return None
-    query_labels = {}
+    query_labels = []
     for key, val in orig_query_labels.items():
-        query_labels[urllib.unquote(key)] = urllib.unquote(val)
+        if isinstance(val, list):
```

Comment:
This PR adds undocumented support to the API for single OR list values in label queries. This should be made explicit in the documentation. Personally I'd prefer to document using array values across the board instead of supporting a union type (string|[]string). dsub would need to fail if it saw >1 value. Arguably this should be something in the capabilities response, whether these kinds of "or" queries are supported.